### PR TITLE
[ADD] mrp_repair_report_customer_lang

### PR DESCRIPTION
**Pull request by eLBati**
By default, repair orders are printed in user language.
This module allows to use partner language as report language,
so that you can send the PDF to your customer.

### CODE REVIEW

**Comment by tafaRU** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Sorry again for disturbing you with my name changes request. I think that a more homogenized name should be _mrp_repair_report_customer_lang_

File: mrp_repair_partner_language_report/views/report_mrp_repair_order.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+<template id="report_mrp_repair_order_translate">
+    <t t-raw="translate_doc(doc_ids[0], doc_model, 'partner_id.lang', 'mrp_repair.report_mrprepairorder')"/>
```
Comment (pedrobaeza) [inline mrp_repair_partner_language_report/views/report_mrp_repair_order.xml]: Maybe you can make here a `t-for` for using this with multi-records.

Comment (eLBati) [inline mrp_repair_partner_language_report/views/report_mrp_repair_order.xml]: @pedrobaeza Due to the structure of `report_mrprepairorder` https://github.com/odoo/odoo/blob/8.0/addons/mrp_repair/views/report_mrprepairorder.xml using `t-for` would generate several calls to `report.html_container`, thus an invalid HTML page.
This is why I added https://github.com/eLBati/manufacture-reporting/blob/mrp_repair_report_partner_lang/mrp_repair_report_customer_lang/README.rst#known-issues--roadmap

**Comment by eLBati** [comment]
@pedrobaeza thanks, module renamed

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Pull request merged**